src/set_theory/surreal/basic.lean

```diff
@@ -286,7 +284,7 @@ instance : ordered_add_comm_group surreal :=
   le_trans          := by { rintros ⟨_⟩ ⟨_⟩ ⟨_⟩, apply @le_trans pgame },
   lt_iff_le_not_le  := by { rintros ⟨_, ox⟩ ⟨_, oy⟩, exact lt_iff_le_not_le },
   le_antisymm       := by { rintros ⟨_⟩ ⟨_⟩ h₁ h₂, exact quotient.sound ⟨h₁, h₂⟩ },
-  add_le_add_left   := by { rintros ⟨_⟩ ⟨_⟩ hx ⟨_⟩, exact @add_le_add_left pgame _ _ _ _ _ hx _ } }
+  add_le_add_left   := by { rintros ⟨_⟩ ⟨_⟩ hx ⟨_⟩, apply @add_le_add_left pgame _ _ _ _ _ hx } }
```

Comment:
On the one hand, you save 2 symbols here. On the other hand, Lean and a human reading the code have to try `term`, then `term _` every time. Not sure what's more important (both aren't important). Feel free to leave as is or revert this line.